### DISCUSSION TITLE
Dotfiles for plugin demo from screenshots

### DISCUSSION THREAD

**Discussion by life00** [huantrinh1802/m_taskwarrior_d.nvim#53] [General]
Hi. It would be nice if @huantrinh1802 you could share your dotfiles for the demo of m_taskwarrior_d.nvim from screenshots. I am especially interested in how you made concealing for todo's and how to change the syntax highlight of `$id{}` and `$query{}`.

Nice plugin btw, perfectly fits my workflow.

**Comment by huantrinh1802** [comment]
Thanks. 

For the todos (or [ ], [x], etc) concealing, I rely on [obsidian.nvim](https://github.com/epwalsh/obsidian.nvim/tree/main). I included my config for obsidian.nvim in the README.

For `$id{}` and `$query{}`, do you mean how to change the icons and colour for the concealing? If yes, it is included in the plugin and unfortunately, I haven't refactored those into config options yet but soon I hope.
